newIDE/app/src/UI/TreeView/index.js

```diff
@@ -285,6 +289,9 @@ const TreeView = <Item: ItemBaseAttributes>(
             name,
             rightComponent,
             rightButton,
+            shouldHideMenuIcon: shouldHideMenuIcon
```

Comment:
A bit confusing to have the boolean and the callback having the same name but ok